openedx/stanford/lms/envs/common.py

```diff
@@ -8,6 +8,12 @@
 # This is actually just a base email.  We'll make it 'noreply+<username>@example.com' to ensure uniqueness
 ANONYMOUS_USER_EMAIL = 'noreply@example.com'
 API_DATE_FORMAT = '%Y-%m-%d'
+
+# Additional queue for grades backfill task (persistent grades).
+GRADES_BACKFILL_QUEUE = 'edx.core.grades_backfill'
+CELERY_QUEUES.update({
+    GRADES_BACKFILL_QUEUE: {},
```

Comment:
Nit, but should this just be a literal string here instead of introducing a variable if it's not used anywhere else?